cfgov/unprocessed/apps/paying-for-college/js/models/financial-model.js

```diff
@@ -167,13 +167,21 @@ const financialModel = {
   _enforceLimits: () => {
     let unsubCap = 0;
     const errors = {};
+    const yearMap = {
```

Comment:
Could this be exported from `debt-calculator.js` and then imported and used in this file so there isn't a redundant data map between the two files?

Reply:
Yes! I'd like to actually refactor this all to be simpler, but that'll have to wait for a future update. Thanks!